### DISCUSSION TITLE
It looks like this IBAN is already in use.

### DISCUSSION THREAD

**Discussion by piotrcichosz** [firefly-iii/firefly-iii#7603] [Firefly III]
I'm getting that when trying to edit existing account (I want to add some note.. )
Can you point me to how/where can I look if that IBAN was used in the past?

**Comment by piotrcichosz** [comment]
ok, I found it in revenue accounts

but there's other error- when editing it ask for opening balance date (since amount is provided), so I added date, saved, opened again edit form - opening balance date is again not set

**Reply by JC5** [reply]
I see what you mean, I'll pick it up!
